File: modul_4/PRAK402-2510817220008-NamiraHumaira.py
batas_maksimal = int(input(""))
for i in range (1, batas_maksimal + 1, 2):
    print(i, end=" ")
print("")
for i in range (batas_maksimal, 1, -1):
    if i % 2 == 0:
        print(i, end=" ")

File: modul_4/PRAK402-2510817220008-NamiraHumaira.c
#include <stdio.h>
int main ()
{
    int batas_maksimal;
    scanf("%d", &batas_maksimal);

    for (int i = 1; i <= batas_maksimal; i+= 2)
    {
        printf("%d ", i);
    }
    printf("\n");
    for (int i = batas_maksimal; i >= 2; i--)
    {
        if(i % 2 == 0)
        {
            printf("%d ", i);
        }
    }
    return 0;
}